spinalcordtoolbox/deepseg/models.py

```diff
@@ -122,7 +122,7 @@
     #       - Binarization is applied within SCT code
     "model_seg_sc_contrast_agnostic_softseg_monai": {
         "url": [
-            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/v2.4/model_soft_bin_20240425-170840.zip"
+            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/v2.5/model_contrast-agnostic_20240930-1002.zip"
```

Comment:
@jcohenadad do you mean instead of `v2.5`, it should be `r20241024`?

Or do you mean the date in the `.zip` filename?

Reply:
i meant the tag of the release should be r20241024 instead of v2.5 as per [our conventions](https://github.com/ivadomed/ivadomed/wiki/repositories#packaging-models)

Reply:
let's do this in another PR-- we are too much pressed with time (https://github.com/spinal-cord-7t/coil-qc-code/issues/139)

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4673